src/modules/songs/forms/AddSongToSet/components/SetSelectionUpcomingSets.tsx

```
@@ -0,0 +1,110 @@
+            </HStack>
+            <Skeleton className="h-5 w-20" />
+          </HStack>
+        </VStack>
+      </SetSelectionSection>
+    );
+  }
+
+  if (!!upcomingSetsQueryError || !upcomingSetsData) {
+    // TODO: how do we better handle this?
+    toast.error(
+      `Ran into an issue when trying to find the upcoming sets${!!upcomingSetsQueryError ? `: ${upcomingSetsQueryError.message}` : ""}`,
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)